components/kyma-environment-broker/README.md

```diff
@@ -25,6 +25,7 @@ KEB binary allows you to override some configuration parameters. You can specify
 | Name | Description | Default value |
 |-----|---------|:--------:|
 | **APP_PORT** | Specifies the port on which the HTTP server listens. | `8080` |
+| **APP_BROKER_DEFAULT_GARDENER_SHOOT_PURPOSE** | Specifies the purpose of the created cluster. The possible values are: `development`, `evaluation`, `production`, `testing`. | `development` |
```

Comment:
The env name is `APP_PROVISIONING_DEFAULT_GARDENER_SHOOT_PURPOSE` now :D Sorry for inconvenience